src/ctelnet.cpp

```diff
@@ -4,7 +4,8 @@
  *   Copyright (C) 2013-2014 by Stephen Lyons - slysven@virginmedia.com    *
  *   Copyright (C) 2014 by Ahmed Charles - acharles@outlook.com            *
  *   Copyright (C) 2015 by Florian Scheel - keneanung@googlemail.com       *
- *   Copyright (C) 2016 by Ian Adkins - ieadkins@gmail.com                 *
+ *   Copyright (C) 2017 by Darksix - https://launchpad.net/~deathlire      *
+ *   Copyright (C) 2017 by RockHound                                       *
```

Comment:
Michael Hupp (Darksix) - darksix@northfire.org
Far as RockHound goes his email is in the other post, not to sure he wants to put a name on it.  Will have to ask again when he's around later.